wordpress-mobile/WordPress-Android
# Feature/support multiple auth headers in gb

This PR addresses changes in GB side to allow multiple headers to be passed to editor's `OkHttpHeaderInterceptor.

GB sitde changes are [here](https://github.com/wordpress-mobile/gutenberg-mobile/pull/2080).

To test:
- Using a private site, navigate to post that contains an image using GB or Aztec.
- Notice that the image is correctly loaded.

PR submission checklist:
